.travis.yml

```diff
@@ -0,0 +1,13 @@
+sudo: required
+services:
+  - docker
+
+before_install:
+  - docker pull fmonteiroctw/mococrw_build_env
+  - docker images
+
+script:
+  - mkdir build
+  - docker run --rm -t -v "$(readlink -f .):/src" -v "$(readlink -f build):/build" --workdir /build fmonteiroctw/mococrw_build_env cmake -DCMAKE_BUILD_TYPE=Coverage -DBUILD_TESTING=On /src
```

Comment:
Can we make `fmonteiroctw/mococrw_build_env` a container that's automatically built from a certain source, so anybody can rebuild it at a later point in time?

Reply:
Ageed. Linked github repository to the dockerhub repository so further builds can be made.

Reply:
Ah, it's only visible with login, so maybe that was there before and I just missed it.